pkg/apis/backupdriver/v1/snapshot.go

```diff
@@ -0,0 +1,148 @@
+package v1
+
+import meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type SnapshotSpec struct {
+	// ResourceHandle refers to a Kubernetes resource, currently a PVC but this may be extended in the future
+	ResourceHandle string
+
+	// The backup repository to snapshot into.  The namespace the Snapshot/PVC lives in must have access to the repository
+	BackupRepository string
```

Comment:
Is this field required?

Reply:
This is how we determine the backup repository for the snapshot.